modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v1/services/Transactions.scala

```diff
@@ -116,6 +123,31 @@ object Transactions {
         }
         .thrushK(trans.xa)
 
+    def getByAddress(
+      address: Address,
+      paging: Paging,
+      concise: Boolean,
+      inclusionHeightRange: InclusionHeightRange
+    ): F[Items[TransactionInfo]] =
+      transactions
+        .countRelatedToAddress(address, inclusionHeightRange.fromHeight, inclusionHeightRange.toHeight)
+        .flatMap { total =>
+          val narrowBy = if (concise) Some(address) else None
+          transactions
+            .streamRelatedToAddress(
+              address,
+              paging.offset,
+              paging.limit,
+              inclusionHeightRange.fromHeight,
+              inclusionHeightRange.toHeight
+            )
+            .chunkN(serviceSettings.chunkSize)
+            .through(makeTransaction(narrowBy))
+            .to[List]
+            .map(Items(_, total))
+        }
+        .thrushK(trans.xa)
```

Comment:
This method heavily duplicates code from `getByAddress`. Why not just adding `InclusionHeightRange` to the old one as an optional constraint?